poh/src/poh_recorder.rs

```diff
@@ -196,6 +196,74 @@ pub enum PohLeaderStatus {
     Reached { poh_slot: Slot, parent_slot: Slot },
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub struct LeaderTickHeights {
+    pub first_tick_height: u64,
+    pub first_tick_reached_at: Option<Instant>,
+    pub target_tick_reached_at: Option<Instant>,
```

Comment:
nit: prefer to keep the original more explicit naming `target_tick_including_grace_ticks_reached_at`